### ISSUE TITLE
browser should not offer to save the key password/passphrase

### ISSUE THREAD

**Issue by m4lvin** [mailvelope/mailvelope#295]
I just created a new key with Mailvelope v0.12.1 in Iceweasel 36.0.1. Right after the key creation starts, the browser offers me to save the password. I find this both irritating and dangerous. Saving the key password/passphrase in the browser password list is as good as having a private key without passphrase.
![mailvelope](https://cloud.githubusercontent.com/assets/2990728/6675361/22359662-cc20-11e4-97f5-29ede5c4e039.png)
Probably this can be prevented with some attribute on the password input field?

**Comment by My1** [comment]
not really, the browsers seem to actively go against the W3C in this one, a new firefox change in the dev version for ff38 for example says "autocomplete=off is no longer supported for username/password fields" mainly because probably that feature is overused
